mkinitcpio

```diff
@@ -211,6 +211,9 @@ build_image() {
         lz4)
             COMPRESSION_OPTIONS+=('-l')
             ;;
+        zstd)
+            COMPRESSION_OPTIONS+=('-19')
```

Comment:
How about  `-T0` (multithreaded compression)?

Reply:
None of the tools used in mkinitcpio are threaded, so probably not

Reply:
If -T0 is faster (should be) and still makes reproducible archives, that's great. On the other hand, I'd say we should use reasonable conservative defaults (matching upstream zstd defaults), and let users tweak further, if they want.

Reply:
It should be reproducible [1]. `-T0` is also used in devtools [2], but admittedly, unlike some packages, initcpios aren't usually large enough for parallel compression to make a big difference.

[1] https://github.com/facebook/zstd/issues/2238
[2] https://github.com/archlinux/devtools/blob/20200407/makepkg-x86_64.conf#L135

Reply:
> I'd say we should use reasonable conservative defaults (matching upstream zstd defaults)

I think `-19` isn't upstream default already otherwise you wouldn't need to pass it explicitly.

Reply:
Debian uses `-19 -T0` for what it's worth: https://salsa.debian.org/kernel-team/initramfs-tools/-/blob/cbbbb1b73e72124ba517325c51e0ea4bd7139104/mkinitramfs#L187

Reply:
They also use pigz for gzip is available, we don't.
Maybe we can just do another PR to switch to multi-threaded tools if possibile?

Reply:
I'm fine with using -T0 right now, as long as it's reproducible (it seems to be). Also, what I meant by -19 upstream is the kernel recommendation to use -19, not zstd's. Anyway, @luca020400, if you want to update this branch to add -T0, I'm fine with it.

Reply:
Done

Reply:
Will it even matter in this case? What's the block size used for -19 compared to the size of the typical initramfs?

Reply:
I think you mean window size? It's 21M. Fallback initramfs can easily be 100+. 
On my machine `-T0` speeds up `mkinitcpio -P` by 35%.

Reply:
The regular initramfs is around that window, but I'd say it still has benefits, even if you, like me, aren't using the fallback initramfs (I'm thinking about disabling it's generation *by default*)